policyengine_us/variables/household/demographic/person/has_special_needs.py

```diff
@@ -0,0 +1,8 @@
+from policyengine_us.model_api import *
+
+
+class has_special_needs(Variable):
```

Comment:
is there a legal code definition? And this specific for this program (are there special rules which make you "special needs" for this program)

Reply:
i found another source that has more details (which is NE specific) so I will move variable location